example/lib/v2.dart

```diff
@@ -0,0 +1,198 @@
+                          responseOutput =
+                              "GetInsightV2 -> ${json.encode(response)}";
+                        });
+                      } catch (e) {
+                        throw e.toString();
+                      }
+                    },
+                    child: const Text('Get Overview Insight'),
+                  ),
+                  const SizedBox(
+                    height: 10.0,
+                  ),
+                  ElevatedButton(
+                    onPressed: () async {
+                      setState(() {
+                        responseOutput = "GetBankInsight -> loading.. ";
+                      });
+                      var response;
+                      try {
+                        response = await FlutterPericulum
+                            .getMobileV2BanksForInsightOverviewKey(
+                                token: "${dotenv.env['tokenKey']}",
+                                insightsOverviewKey: '2');
+                        setState(() {
+                          responseOutput =
+                              "GetBankInsight -> ${json.encode(response)}";
+                        });
+                      } catch (e) {
+                        throw e.toString();
+                      }
+                    },
+                    child: const Text('Get Banks Insight'),
+                  ),
+                  const SizedBox(
+                    height: 10.0,
+                  ),
+                  ElevatedButton(
+                    onPressed: () async {
+                      setState(() {
+                        responseOutput = "GetLendersOverview -> loading.. ";
+                      });
+                      var response;
+                      try {
+                        response = await FlutterPericulum
+                            .getMobileV2LendersForInsightOverviewKey(
+                                token: "${dotenv.env['tokenKey']}",
+                                insightsOverviewKey: '2');
+                        setState(() {
+                          responseOutput =
+                              "GetLendersOverview -> ${json.encode(response)}";
+                        });
+                      } catch (e) {
+                        throw e.toString();
+                      }
+                    },
+                    child: const Text('Get Lenders Insight'),
+                  ),
+                  const SizedBox(
+                    height: 10.0,
+                  ),
+                  ElevatedButton(
+                    onPressed: () async {
+                      setState(() {
+                        responseOutput = "GetTransactionInsights -> loading.. ";
+                      });
+                      var response;
+                      try {
+                        response = await FlutterPericulum
+                            .getMobileV2TransactionsByBankAddressAndAccountNumberForInsightOverviewKey(
```

Comment:
can you please rename method to "GetSMSTransactionsByBankKeyAndAccountKeyForInsightsOverviewKeyAsync" ?